build/pmm-client-docker/Dockerfile

```diff
@@ -1,20 +1,25 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi7/ubi
 RUN yum -y install iproute nc \
     && rm -rf /var/cache/yum
 
 COPY pmm2-client.tar.gz /tmp/pmm2-client.tar.gz
+RUN groupadd -g 1002 pmm-agent
+RUN useradd -u 1002 -r -g pmm-agent -s /sbin/nologin \
+            -d /usr/local/percona/pmm2 \
+            -c "Default Application User" pmm-agent
```

Comment:
I think we can make a better comment. 😄 Something like "PMM 2.x Client User".

Reply:
done